services/credential-server/src/agent.ts

```diff
@@ -189,6 +185,9 @@ class Agent {
 
     const grantNotification = (await getHolderNotifications()).notes[0];
 
+    // resolve schema
+    await this.signifyApi.resolveOobi(`${config.oobiEndpoint}/oobi/${Agent.QVI_SCHEMA_SAID}`);
```

Comment:
why do we need to resolve this here?

Reply:
In the server we can issue LE (legal entity) credentials. These are chained to QVI credentials (ACDCs can be chained...). So on start-up, we are creating a Signify client for issuer and a Signify client for holder and issuing a QVI credential to the holder.

The issuer resolves the schema when issuing the credential, but this line was missing for the holder to resolve the schema. There were loads of errors in KERIA until I added this line

Reply:
There are 2 Signify clients (which means 2 agents in KERIA) - each are resolving it once. So yes it's 2 times overall but it's only once per client.

Reply:
ah, sorry. I thought it was the same client.